v1/src/main/java/com/google/cloud/teleport/spanner/ddl/InformationSchemaScanner.java

```diff
@@ -945,8 +978,8 @@ private void listChangeStreams(Ddl.Builder builder) {
                     + " AND cst.table_catalog = csc.table_catalog"
                     + " AND cst.table_schema = csc.table_schema"
                     + " AND cst.table_name = csc.table_name"
-                    + " GROUP BY cs.change_stream_name, cs.all, cst.table_name, cst.all_columns"
-                    + " ORDER BY cs.change_stream_name, cs.all, cst.table_name"));
+                    + " GROUP BY cs.change_stream_name, cs.all, cst.table_schema, cst.table_name, cst.all_columns"
+                    + " ORDER BY cs.change_stream_name, cs.all, cst.table_schema, cst.table_name"));
```

Comment:
I see many queries have ` WHERE t.table_schema NOT IN ('INFORMATION_SCHEMA', 'SPANNER_SYS')` where clause added. But some of the queries, like the above don't have this new where clause. I hope they are correct and are tested.

Reply:
ype, because change stream can not show up in system catalog, that is why it works. 

But you are right, to keep consistent, and the import/export semantic should be: List user created/creatable objects, no matter what object it is. Let me think about it to see how to improve/enhance.